src/pages/tx/ExecutionInfo.tsx

```diff
@@ -10,15 +10,15 @@ interface Props {
 }
 
 export function ExecutionInfo({ tx }: Props): JSX.Element {
-  const time = fromRfc3339(tx.timestamp);
+  // const time = fromRfc3339(tx.timestamp);
```

Comment:
Is this not available in stargate?

Reply:
It's not in our `IndexedTx` type: https://github.com/cosmos/cosmjs/blob/master/packages/stargate/src/stargateclient.ts#L34-L42
Maybe it's in the response but we haven't added it so far?

Reply:
Would be good to find out. We might need to get the information via the block time if the backend does not provide it for us. It is possible that the REST server did this for us in launchpad times. The time of the block is the only transaction time that exists in the data storage.

Reply:
But this should not be blocking the rest of this work to be merged.

Reply:
Just checked and this information doesn't seem to be visible in the response.

Reply:
Thanks, let's tackle that ndependently: https://github.com/CosmWasm/code-explorer/issues/35